tests/datetime_test.py

```diff
@@ -34,6 +48,33 @@ def setUp(self):
         self.tdvec2 = ak.Timedelta(ak.ones(100, dtype=ak.int64), unit="s")
         self.onesecond = pd.Timedelta(1, unit="s")
 
+    def test_timedelta(self):
+        ak_df = build_ak_df()
```

Comment:
I think we should probably just build this DataFrame using pandas since we are testing against it. Otherwise, we are still testing the functionality against itself.

Reply:
Now using pandas dataframe